src/SerialLoops.Lib/Config.cs

```diff
@@ -102,7 +104,9 @@ private static Config GetDefault(ILogger log)
                 EmulatorPath = emulatorPath,
                 AutoReopenLastProject = true,
                 RememberProjectWorkspace = true,
-                RemoveMissingProjects = false
+                RemoveMissingProjects = false,
+                CheckForUpdates = true,
+                PreReleaseChannel = true
```

Comment:
Set to false by default

Reply:
also, the nittiest of nits, but should be
```suggestion
                PreReleaseChannel = false,
```

the comma makes it so next time you update the list, that line doesn't get included in the diff